Backpack/Card/Classes/BPKCard.h

```diff
@@ -44,6 +57,14 @@ NS_SWIFT_NAME(Card) IB_DESIGNABLE @interface BPKCard: UIControl
  */
 - (instancetype)initWithPadded:(BOOL)padded NS_DESIGNATED_INITIALIZER;
```

Comment:
This should no longer be marked as designated since the new one is more configurable. This should call the new one under the hood.